data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg

```diff
@@ -433,7 +433,7 @@
 
         [message]
             speaker="Ro'Sothian"
-            message= _ "Huh?! He killed me!"
+            message= _ "Why not? He killed me!"
```

Comment:
I like the original more.

Reply:
I like the new ones, if other people agree with you however, I'll change it.

Reply:
Please revert to original, the "Huh?" feels more indignant to me.